src/passwd.c

```diff
@@ -393,7 +393,7 @@ static void check_password (const struct passwd *pw, const struct spwd *sp)
 	 * PAM) was specified, do nothing. --marekm
 	 */
 	if (kflg && (0 == exp_status)) {
-		exit (E_SUCCESS);
+		fail_exit(E_SUCCESS, process_selinux);
```

Comment:
Should the fail_exit changes go in a separate commit?

Reply:
Adjusted